### ISSUE TITLE
can't read "::destination_parameters__pruning_mode"

### ISSUE THREAD

**Issue by tabernarious** [tabernarious/f5-automated-backup-iapp#14]
Need to fix error: can't read "::destination_parameters__pruning_mode"

Here is the original post by keithhubb:

----------
I'm using v3.1.3, and getting this message: 

script did not successfully complete: (can't read "::destination_parameters__pruning_mode": no such variable
while executing
"if { $::destination_parameters__pruning_mode eq "Only Prune iApp-Generated Archives" } {
set pruning_suffix $::destination_par..."
invoked from within
"if { $freq != "Disable" } {
Ensure a default $filename_format is set
if { $::destination_parameters__filenam..." line:44)

I Fixed it by changing line 54 from this:

if { $::destination_parameters__pruning_mode eq "Only Prune iApp-Generated Archives" } {

To this:

if { [info exists ::destination_parameters__pruning_mode] && ($::destination_parameters__pruning_mode eq "Only Prune iApp-Generated Archives") } {

Basically validating the variable exists as a condition to the 'if' statement.

**Comment by tabernarious** [comment]
I have verified and tested the suggested fix which will be included in v3.1.4
